src/locale/cs/LC_MESSAGES/django.po

```diff
@@ -7,7 +7,7 @@ msgid ""
 msgstr ""
```

Comment:
Don't bother with translations for now, they often cause merge conflicts and we'll tune them later anyway. But thanks for being exhaustive:)